src/item_factory.cpp

```diff
@@ -2458,6 +2458,7 @@ void Item_factory::add_entry( Item_group &ig, JsonObject &obj )
                                              probability ) );
     }
     if( !sptr ) {
+        debugmsg( "Failed to load an entry in an item group." );
```

Comment:
I suggest the following:

```suggestion
        debugmsg( "Failed to load an entry in an item group (item=[%s],group=[%s]).\nContext:\n%s",
                  obj.get_string( "item", "" ), obj.get_string( "group", "" ), obj.str() );
```

![image](https://user-images.githubusercontent.com/16213433/59961335-fe344600-94de-11e9-97ae-a2308896dd16.png)

Reply:
We should probably also expand it with

```C++
if( !obj.has_member( "item" ) && !obj.has_member( "group" ) ) {
    debugmsg( "Expected either `item` or `group`, but found none!\nContext:\n%s", obj.str() );
}
if( obj.has_member( "item" ) && obj.has_member( "group" ) ) {
    debugmsg( "Expected either `item` or `group`, but found both!\nContext:\n%s", obj.str() );
}
```